src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/GithubPullRequestDecorator.java

```diff
@@ -102,7 +102,7 @@ private GHPullRequest createCheckRun(GHRepository repository, AnalysisDetails an
             );
         }
 
-        repository.createCheckRun(String.format("%s Sonarqube Results", analysisDetails.getAnalysisProjectName()), analysisDetails.getCommitSha())
+        repository.createCheckRun("SonarQube Code Analysis", analysisDetails.getCommitSha())
```

Comment:
I think we need to cater for the scenario where a Monorepo is being used, so have some logic in here that includes the project name when Monorepo is set?